src/toolchain_cmd.go

```diff
@@ -73,6 +73,15 @@ func init() {
 		log.Fatal(err)
 	}
 
+	_, err = c.AddCommand("temp-dir",
+		"get toolchain's temp dir",
+		"Get toolchain's temp directory. Creates it if it doesn't exists.",
```

Comment:
- [ ] I'm not sure what standard we're following for grammar, but these two lines should be "get a toolchain's temp dir" and "Get a toolchain's temporary directory. Create it if it doesn't exist."
The second line is seen as documentation, so we try not to use abbreviations like "temp".